### PR TITLE
release: tag: tag kernel build

### PR DESCRIPTION
**Pull request by jcvenegas**
On each release we want to know the kernel config what was used.
Lets create a tag ${kata_verson}-kernel-config.

### CODE REVIEW

File: release/tag_repos.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
```
Comment (grahamwhaley) [inline release/tag_repos.sh]: Just to check - I think enabling the debugging output was deliberate, yes? (as you also added the /dev/null redirects etc).

**Comment by grahamwhaley** [comment]
But, the CI failed due to an existing tag it seems - something we need to fix and/or cater for?
```
+ git tag -a 1.1.0-kernel-config -s -m 'Kata Containers release 1.1.0-kernel-config'
fatal: tag '1.1.0-kernel-config' already exists
+ finish
```

File: release/tag_repos.sh [hunk 2]
```diff
@@ -85,25 +86,30 @@ tag_repos() {
...
+		pushd "${repo}" >> /dev/null
 		git remote set-url --push origin "git@github.com:${OWNER}/${repo}.git"
 		git fetch origin --tags
 		if git rev-parse -q --verify "refs/tags/${kata_version}"; then
 			info "$repo already has tag "
 		else
-			info "Creating tag ${kata_version} for ${repo}"
-			git tag -a "${kata_version}" -s -m "${PROJECT} release ${kata_version}"
+			tag="$kata_version"
+			[[ "packaging" == "${repo}" ]] && tag="${tag}-kernel-config"
```
Comment (jodh-intel) [inline release/tag_repos.sh]: Nit: the double-square brackets are only (?) required for magic pattern matching I think so I'd use single ones for clarity generally.

File: release/tag_repos.sh (hunk 1 above)
Comment (jcvenegas) [reply release/tag_repos.sh]: yes, removing now.

**Pull request merged**